src/api/types/generated-schema.ts

```diff
@@ -3822,7 +3838,7 @@ export interface components {
       rare?: boolean;
       scientificName: string;
       /** @enum {string} */
-      seedStorageBehavior?: "Orthodox" | "Recalcitrant" | "Intermediate" | "Unknown" | "Likely Orthodox" | "Likely Recalcitrant" | "Likely Intermediate";
+      seedStorageBehavior?: "Orthodox" | "Recalcitrant" | "Intermediate" | "Unknown" | "Likely Orthodox" | "Likely Recalcitrant" | "Likely Intermediate" | "Intermediate - Cool Temperature Sensitive" | "Intermediate - Partial Desiccation Tolerant" | "Intermediate - Short Lived";
```

Comment:
I believe these changes were already made by Steve, can you sync this branch with main?